### PR TITLE
Update readme with basic usage and contacts

### PR DESCRIPTION
**Pull request by manics**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,41 @@ OMERO.searcher provides content-based image search for
...
+This is beta quality software.
+
+* Features calculated by OMERO.searcher are stored independently of the
+OMERO repository.
+* Features should be calculated by a group owner to avoid problems with
+permissions.
+* The OMERO.searcher web-app requires direct access to the features store.
+* Multiple simultaneous feature calculation processes will conflict with
+each other.
+* If images are modified, moved or deleted the feature Content database
```
Comment (mtbc) [inline README.md]: maybe +then?

File: README.md [hunk 2]
```diff
@@ -5,3 +5,41 @@ OMERO.searcher provides content-based image search for
...
+will become desynchronised. If errors occur when performing a search it may
```
Comment (mtbc) [inline README.md]: US spelling

**Comment by mtbc** [comment]
Looks good overall.

**Comment by manics** [comment]
@mtbc How's this?

**Comment by mtbc** [comment]
Great, thank you, good to merge.

**Pull request merged**